verify_circuit_tests/src/lib.rs

```diff
@@ -1,22 +1,39 @@
-use ark_ec::{short_weierstrass_jacobian::GroupAffine, AffineCurve};
+use ark_ec::AffineCurve;
 use ark_ff::{One, PrimeField};
 use ark_poly::domain::EvaluationDomain;
 use kimchi::{
     curve::KimchiCurve,
     error::VerifyError,
-    mina_curves::pasta::{Fq, PallasParameters},
+    mina_curves::pasta::{Fq, Pallas, PallasParameters},
     mina_poseidon::{
         constants::PlonkSpongeConstantsKimchi,
         sponge::{DefaultFqSponge, DefaultFrSponge},
     },
     o1_utils::FieldHelpers,
     poly_commitment::PolyComm,
-    proof::{LookupEvaluations, PointEvaluations, ProofEvaluations, ProverProof},
+    proof::{
+        LookupCommitments, LookupEvaluations, PointEvaluations, ProofEvaluations,
+        ProverCommitments, ProverProof, RecursionChallenge,
+    },
     verifier_index::VerifierIndex,
 };
 use serde::Serialize;
 
-pub type PallasGroup = GroupAffine<PallasParameters>;
+pub type PallasScalar = <Pallas as AffineCurve>::ScalarField;
+pub type PallasPointEvals = PointEvaluations<Vec<PallasScalar>>;
+
+pub struct SerializablePallasScalar {
+    element: PallasScalar,
+}
```

Comment:
It's preferable to use a _tuple struct_ for these cases.

Reply:
thank you! fixed